File: year2/systems-design-implementation/Lab10/SwimClientSharp/SwimClientSharp/Proba.cs
using System;

namespace SwimClientSharp
{
    public class Proba
    {
        public Proba()
        {
        }

        public Proba(int id, int distanta, string stil)
        {
            this.id = id;
            this.distanta = distanta;
            this.stil = stil;
        }
        
        public Proba(int distanta, string stil)
        {
            this.distanta = distanta;
            this.stil = stil;
        }

        public int id { get; set; }
        public String stil { get; set; }
        public int distanta { get; set; }

        public override string ToString()
        {
            return "[" + id + "] " + stil + " " + distanta + "m";
        }
    }
}

File: year2/systems-design-implementation/Lab10/SwimModel/src/main/java/domain/Proba.java
package domain;

import java.io.Serializable;

public class Proba implements Serializable {

    private Integer id;
    private Integer distanta;
    private String stil;

    public Proba() {
    }

    public Proba(Integer id, Integer distanta, String stil) {
        this.id = id;
        this.distanta = distanta;
        this.stil = stil;
    }

    public Proba(Integer distanta, String stil) {
        this.distanta = distanta;
        this.stil = stil;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getDistanta() {
        return distanta;
    }

    public void setDistanta(Integer distanta) {
        this.distanta = distanta;
    }

    public String getStil() {
        return stil;
    }

    public void setStil(String stil) {
        this.stil = stil;
    }

    @Override
    public String toString() {
        return "[" + id + "] " + stil + " " + distanta + "m";
    }
}
